src/main/java/gov/nci/ppe/services/impl/AmazonS3ServiceImpl.java

```diff
@@ -461,36 +462,53 @@ public Code getCodeforFileType(String fileTypeCodeName) {
 		return codeService.getCode(fileTypeCodeName);
 	}
 
+	
 	/**
-	 * Method to send out email to patients, their providers and CRC when a
-	 * biomarker report is uploaded by the admin
+	 * Helper method to send out emails after file upload
 	 * 
-	 * @param patient - An object of Participant
-	 * @param admin   - An object of User
-	 * @return
+	 * @param emailIds        - list of emailIds to mail to
+	 * @param patientFullName - Full Name of the Patient (FirstName LastName)
+	 * @param actionFor       - File type Uploaded
 	 */
-	private String sendEmailToAdminAfterFileUpload(Participant participant, String emailId) {
-
-		DateTimeFormatter dtf = DateTimeFormatter.ofPattern("yyyy/MM/dd-HH:mm:ss");
-		LocalDateTime now = LocalDateTime.now();
-		dtf.format(now);
-		String[] dateTime = StringUtils.split(dtf.format(now), "-");
-
-		/* Replace the variables in the EmailBody */
-		String replaceStringWith[] = { participant.getFirstName(), participant.getLastName(),
-				participant.getPatientId(), dateTime[0], dateTime[1] };
-		String replaceThisString[] = { "%{FirstName}", "%{LastName}", "%{UserGUID}", "%{date}", "%{time}" };
-		String updatedTextBody = StringUtils.replaceEach(emailServiceConfig.getEmailTextBodyForAdmin(),
-				replaceThisString, replaceStringWith);
-
-		/* Replace the variables in the Subject Line */
-		String replaceSubjectStringWith[] = { participant.getPatientId() };
-		String replaceThisStringForSubject[] = { "%{UserGUID}" };
-		String subject = StringUtils.replaceEach(emailServiceConfig.getEmailSubjectForAdmin(),
-				replaceThisStringForSubject, replaceSubjectStringWith);
-
-		return emailLogService.sendEmailNotification(emailId, emailServiceConfig.getSenderEmailAddress(), subject,
-				updatedTextBody, updatedTextBody);
-	}
+	private void sendEmailToCRCAndProvidersAfterFileUpload(Map<String, String> emailIds, String patientFullName, String patientId, String actionFor) {
+		if (StringUtils.isNotBlank(actionFor)
+				&& FileType.PPE_FILETYPE_ECONSENT_FORM.getFileType().equalsIgnoreCase(actionFor)) {
+			emailIds.forEach((name, emailId) -> {
+				emailLogService.sendEmailToCRCAndProvidersAfterUploadingBioMarkerReport(name, emailId, patientFullName,patientId);
```

Comment:
This should be the method for eConsent NOT biomarker

Reply:
Will make these changes as part of another commit / story